README.rst

```diff
@@ -9,6 +9,10 @@ celiagg
    :target: https://travis-ci.org/celiagg/celiagg
    :alt: Build status
 
+.. image:: https://ci.appveyor.com/api/projects/status/1d7wb65intsxrn90
+   :target: https://ci.appveyor.com/project/celiagg/celiagg
```

Comment:
Sorry, just noticed this should be: https://ci.appveyor.com/project/jwiggins/celiagg